make.cmd

```diff
@@ -0,0 +1,8 @@
+@echo off
+
+FOR /F "tokens=1,2 delims==" %%A IN (mod.config) DO (set %%A=%%B)
+
+set MOD_SEARCH_PATHS=./mods,%~dp0mods
+
+:start
+powershell -NoProfile -ExecutionPolicy Unrestricted -File make.ps1 %*
```

Comment:
This gives a really obnoxious prompt every time you run it if the mod template repo was downloaded as a zip instead of cloned:
![screen shot 2017-05-13 at 11 50 15](https://cloud.githubusercontent.com/assets/167819/26024849/9c34f960-37d2-11e7-95fb-a77d2c1c0ff0.png)

Can we make the script unblock itself if necessary on first run?